src/client/components/features/Stages3eEt2de/Rechercher/FormulaireRecherche/ListeResultatsStage3eEt2de.tsx

```diff
@@ -61,6 +61,7 @@ function ResultatStage3eEt2de(stage3eEt2de: Stage3eEt2de) {
 				sousTitreOffre={<>
 					<p>{stage3eEt2de.domaine}</p>
 					<p>{stage3eEt2de.adresse.rueEtNumero}, {stage3eEt2de.adresse.codePostal} {stage3eEt2de.adresse.ville}</p>
+					<p>{stage3eEt2de.appellationLibelle.map((appellationLibelle) => appellationLibelle).join(', ')}</p>
```

Comment:
Ça me chagrinait de ne pas voir de liste alors j'ai testé avec 😄 
Il faut un peu de CSS mais voici la proposition : 
```suggestion
					<ul className={styles.listeMetier}>{stage3eEt2de.appellationLibelle.map((appellationLibelle) => <li key={appellationLibelle}><p>{appellationLibelle}</p></li>)}</ul>
```

CSS : 
```css
.listeMetier {
  list-style-type: none;

  > li {
    display: inline;

    > p {
      display: inline;
    }
  }

  > li::before {
    content: ', ';
  }
  > li:first-of-type::before{
    content: none;
  }
}

```

Reply:
😭 oke bien vu 👍